docs/guides/transaction_request_flow.md

```diff
@@ -0,0 +1,110 @@
+# Transaction Request Flow
+
+## Introduction
+
+Transaction requests are a way to create pre-transactions with only one side (either sender or receiver). That pre-transaction will become a full transaction once the other side has consumed it.
+
+
+Here is a quick example to give you a better idea of how it works:
+
+1. Alice generates a transaction request to receive 10 OMG. The transaction ID can be embedded in a QR code to provide a better user experience.
+2. Alice shows her QR code to Bob.
+3. Bob scans it, see the transaction request, updates some parts of it if needed, finalizes it and sends 10 OMG to Alice.
+
+That's the simple version. Transaction requests come with a bunch of options to make them as flexible as possible.
+
+## A look at transaction requests
+
+When creating transaction requests, a certain number of fields are optional but allow the configuration of the request. Walking through those fields is a good way to get a better understanding.
+
+See [the Swagger doc](https://ewallet.demo.omisego.io/api/docs.ui#/TransactionRequest/transaction_request_create) for more details.
+
+Here is the model attributes when creating transaction requests:
+
+```
+type:
+  type: string
+  enum:
+    - send
+    - receive
+token_id:
+  type: string
+  description: "The minted token ID to use for the transaction."
+amount:
+  type: integer
+  default: null
+  description:
+    "The amount to transfer. If not specified, the consumption will need to set
+     the amount."
+correlation_id:
+  type: string
+  default: null
+  description: "Optional unique identifier to tie a request with your system."
+account_id:
+  type: string
+  default: The owner of the given address.
+provider_user_id:
+  type: string
+  default: The owner of the given address.
```

Comment:
Not sure if this is intentional, is it description instead?

Also would be nice if it says either account_id or provider_user_id needs to be filled.